hebrew/sane-option/fresh-start.md

```diff
@@ -1,3 +1,7 @@
 אתה מתעורר בבוקר, ורואה קירות עשויים ממרשמלו.
 
-ברגע הראשון, לא ברור לך מה לעשות, אבל אתה מתעשת ומחליט לעשות משהו.
\ No newline at end of file
+ברגע הראשון, לא ברור לך מה לעשות, אבל אתה מתעשת ומחליט לעשות משהו.
+
+[מתחיל לאכול את הקירות](eating_marshmelow/eating.md)
+
+[בורח מהבית](run2street/run2dstreet.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.